src/renderers/shared/fiber/ReactFiberErrorLogger.js

```diff
@@ -33,6 +33,9 @@ function logCapturedError(capturedError : CapturedError) : void {
       ? `React caught an error thrown by ${componentName}.`
       : 'React caught an error thrown by one of your components.';
 
+    const callStack = error.stack;
+    const trimmedCallStack = callStack.substring(callStack.indexOf('\n'));
```

Comment:
This is not consistent between browsers.
For example Firefox or Safari don’t seem to put the message in stack.

I suggest checking if the first line of the stack ends with error message. If it does, cut it out, otherwise keep it.